up42/jobcollection.py

```diff
@@ -18,9 +23,67 @@ def __init__(self, auth: Auth, project_id: str, jobs: List[Job]):
         self.auth = auth
         self.project_id = project_id
         self.jobs = jobs
+        if jobs is not None:
+            self.jobs_id = [job.job_id for job in jobs]
+        else:
+            self.jobs_id = None
 
     def __repr__(self):
         return (
             f"JobCollection(jobs={self.jobs}, project_id={self.project_id}, "
             f"auth={self.auth})"
         )
+
+    # TODO: Maybe add _jobs_info method?
+    # TODO: Maybe add _jobs_status method?
+
+    def _get_download_url(self, job_id) -> str:
+        url = (
+            f"{self.auth._endpoint()}/projects/{self.project_id}/jobs/{job_id}"
+            f"/downloads/results/"
+        )
+        response_json = self.auth._request(request_type="GET", url=url)
+        download_url = response_json["data"]["url"]
+        return download_url
+
+    def download_parallel_results(
```

Comment:
```suggestion
    def download_results(
```
As syntax of Job() and JobTask()